app/enterprise/changelog.md

```diff
@@ -225,6 +225,12 @@ Kong now ensures targets with a weight of 0 are displayed in the Admin API.
   that extend `base_plugin.lua` will continue to work until v3.0.x but should be updated to the
   newer, simplier [pattern](/enterprise/2.4.x/plugin-development/custom-logic).
 
+### Known issues
+The [MTLS Authentication](/hub/kong-inc/mtls-auth) plugin is incompatible with Kong Gateway v2.4.1.0.
+When making a call using the MTLS Authentication plugin, instead of a successful connection, users
```

Comment:
MTLS is typically abbreviated as "mTLS" with the lower-case m

Reply:
ah, ok, will fix